clients/src/main/java/org/apache/kafka/clients/consumer/RoundRobinAssignor.java

```diff
@@ -36,6 +36,18 @@
  * The assignment will be:
  * C0: [t0p0, t0p2, t1p1]
  * C1: [t0p1, t1p0, t1p2]
+ *
+ * In case that the subscriptions are different across consumer instances, each consumer only gets
```

Comment:
Not sure this really clarifies the process. The sentence "The assignment of topic partitions will be assigned to consumers that subscribed to the topics in a round robin fashion." doesn't add any detail. I'd say something like "When subscriptions differ across consumer instances, the assignment process still considers each consumer instance in round robin fashion but skips over an instance if it is not subscribed to the topic. Unlike the case when subscriptions are identical, this can result in imbalanced assignments."  and then include the example you provided here.